.github/workflows/build-docker-images.yml

```diff
@@ -22,6 +22,16 @@ jobs:
     name: "Latest PyTorch + TensorFlow [dev]"
     runs-on: ubuntu-latest
     steps:
+      - name: Cleanup disk
+        run: |
+          sudo ls -l /usr/local/lib/
+          sudo ls -l /usr/share/
+          sudo du -sh /usr/local/lib/
+          sudo du -sh /usr/share/
+          sudo rm -rf /usr/local/lib/android
+          sudo rm -rf /usr/share/dotnet
+          sudo du -sh /usr/local/lib/
+          sudo du -sh /usr/share/
```

Comment:
When I tried to `install 2.0.0`, the runner ran out of disk. This was the way that fixed the problem.